daemon/daemon_unix.go

```diff
@@ -1190,7 +1190,9 @@ func (daemon *Daemon) initCgroupsPath(path string) error {
 		return nil
 	}
 
-	daemon.initCgroupsPath(filepath.Dir(path))
```

Comment:
Yes, from what I could gather from the original code.

Also, reading the original PR, the following is in the description:
>You will also need to ensure that the system and all parent cgroups have values set for the period and runtime as this limits what the children can be set to.